tests/logprob/test_order.py

```diff
@@ -147,3 +147,25 @@ def test_max_logprob(shape, value, axis):
         (x_max_logprob.eval({x_max_value: test_value})),
         rtol=1e-06,
     )
+
+
+def test_max_discrete():
+    x = pm.DiscreteUniform.dist(0, 1, size=(3,))
+    x.name = "x"
+    x_max = pt.max(x, axis=-1)
+    x_max_value = pt.scalar("x_max_value")
+    x_max_logprob = logp(x_max, x_max_value)
+
+    discrete_logprob = _logprob_helper(x, x_max_value)
+    discrete_logcdf = _logcdf_helper(x, x_max_value)
+    discrete_logcdf_prev = _logcdf_helper(x, x_max_value - 1)
+    n = x.size
+    discrete_logprob = pt.log((pt.exp(discrete_logcdf)) ** n - (pt.exp(discrete_logcdf_prev)) ** n)
```

Comment:
Just compare with pre-computed value, or even better, compare with scipy Poisson

Reply:
And test max = 0